main.go

```diff
@@ -42,6 +42,14 @@ type flagVal struct {
 	OverWrite_flag bool
 	Instance_flag  string
 	Verbose_flag   bool
+	File_flag      bool
+}
+
+// contains local and server paths
+type pathVal struct {
+	RootWorkingDirectoryLocal  string
+	RootWorkingDirectoryServer string
+	StartingPathServer         string
```

Comment:
Okay, so how do you plan to handle that? Refactor to remove it? What was your intention when you created it?